src/EventGridEmulator/Program.cs

```diff
@@ -22,7 +24,15 @@
 
 builder.Services.AddOptions<TopicOptions>().BindConfiguration(string.Empty);
 builder.Services.AddSingleton<SubscriberPolicyHttpMessageHandler>();
-builder.Services.AddHttpClient(SubscriberConstants.HttpClientName, SubscriberConstants.ConfigureHttpClient).AddHttpMessageHandler<SubscriberPolicyHttpMessageHandler>();
+builder.Services.AddHttpClient(SubscriberConstants.HttpClientName, SubscriberConstants.ConfigureHttpClient)
+    .AddHttpMessageHandler<SubscriberPolicyHttpMessageHandler>()
+    .ConfigurePrimaryHttpMessageHandler(handler => new HttpClientHandler
+    {
+        ClientCertificateOptions = ClientCertificateOption.Manual,
+        ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator,
```

Comment:
Due to the fact that this kind of code, where we disable certificate validation, could be considered as suspicious and eventually dangerous, can we explain why we do this?

Reply:
Maybe we can actually create a dedicated class inheriting from HttpClientHandler, with a name and documentation that better shows its intent?